p2p/module.go

```diff
@@ -127,6 +169,26 @@ func (m *p2pModule) Start() error {
 	return nil
 }
 
+func (m *p2pModule) getAddrBookPerHeight(height uint64) (typesP2P.AddrBook, error) {
```

Comment:
Can you rename this to `getStakedAddrBookAtHeight`.

The address book P2P has will be greater (i.e. non-staked nodes), but this what we're going to be using for RainTree (i.e. structured gossip).